core/Interfaces.carp

```diff
@@ -4,6 +4,8 @@
 ;;(definterface copy (λ [&a] a))
 ;;(definterface str (λ [a] String))
 
+(definterface coerce (λ [a] b))
```

Comment:
This is only needed if there are different implementations for this for different types. Is that going to be how this is used? My understanding was that `Unsafe.coerce` kind of stood on its own.

Reply:
Oh, right. Fixed.